tests/core/creators/manual/detachedconfig.html

```diff
@@ -0,0 +1,31 @@
+<button id="attachButton">Attach editor to parent</button>
```

Comment:
We are testing config callback here, so let's take it into account in test name.